src/dune_rules/compilation_context.ml

```diff
@@ -208,12 +208,15 @@ let create ~super_context ~scope ~expander ~obj_dir ~modules ~flags
   ; loc
   }
 
-let for_alias_module t alias_module =
+let for_alias_module t ?flags alias_module =
```

Comment:
```suggestion
let for_alias_module t ?(with_flags=false) alias_module =
```

Reply:
I think it'd be better if this didn't take flags from the outside, since the only use case is to use the flags from the compilation context itself.